app/start.sh

```diff
@@ -12,4 +12,7 @@ pip install -r requirements.txt
 pip install -r ./drucker-grpc-proto/requirements.txt
 sh ./drucker-grpc-proto/run_codegen.sh
 
+python manage.py db init
+python manage.py db migrate
+python manage.py db upgrade
```

Comment:
I think `migrate` and `upgrade` should be done only when DB schemas change, not at every start.
How about executing these 2 actions according to some bash command parameter?

Reply:
I want to run `migrate` and `upgrade` automatically if DB schema are changed.
How should we handle this?